test/k8sT/Services.go

```diff
@@ -645,6 +645,14 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 				testExternalTrafficPolicyLocal(kubectl, ni)
 			})
 
+			// Explicitly test netfilter compat mode with kube-proxy.
+			It("with reverse NAT at host netfilterCompatMode=true", func() {
+				DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
+					"netfilterCompatMode": "true",
+				})
+				testNodePort(kubectl, ni, false, false, false, 0)
+			})
+
```

Comment:
Should this new section be below the "It" section below? Otherwise we will have reconfigured Cilium for the netfilter compatible mode and then we run testNodePort(...) again on line 657 with the same settings, meaning no additional coverage from the extra test?